bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -0,0 +1,81 @@
+let string44 = NSLocalizedString("Empty", comment: "")
+let string45 = NSLocalizedString("Unsupported", comment: "")
+let string46 = NSLocalizedString("Page break block. %s", comment: "translators: accessibility text. %s: Page break text.")
+let string47 = NSLocalizedString("Double tap to edit this value", comment: "translators: accessibility text")
+let string48 = NSLocalizedString("Current value is %s", comment: "")
+let string49 = NSLocalizedString("Double tap to change the value using slider", comment: "translators: accessibility text (hint for focusing a slider)")
+let string50 = NSLocalizedString("Double tap to toggle setting", comment: "translators: accessibility text (hint for switches)")
+let string51 = NSLocalizedString("Content…", comment: "")
+let string52 = NSLocalizedString("Reset Block", comment: "")
+let string53 = NSLocalizedString("Add a description", comment: "")
+let string54 = NSLocalizedString("Add annotation", comment: "")
+let string55 = NSLocalizedString("Remove annotations", comment: "")
+let string56 = NSLocalizedString("Annotations Sidebar", comment: "")
+let string57 = NSLocalizedString("My Document Setting Panel", comment: "")
+let string58 = NSLocalizedString("My post status info", comment: "")
+let string59 = NSLocalizedString("Here is the panel content!", comment: "")
+let string60 = NSLocalizedString("My pre publish panel", comment: "")
+let string61 = NSLocalizedString("My post publish panel", comment: "")
+let string62 = NSLocalizedString("Title:", comment: "")
+let string63 = NSLocalizedString("Sidebar title plugin", comment: "")
+let string64 = NSLocalizedString("Double tap to undo last change", comment: "")
+let string65 = NSLocalizedString("Double tap to redo last change", comment: "")
+let string66 = NSLocalizedString("Hide keyboard", comment: "")
+let string67 = NSLocalizedString("Tap to hide the keyboard", comment: "")
+let string68 = NSLocalizedString("Show section", comment: "translators: Checkbox toggle label")
+let string69 = NSLocalizedString("Post title. Empty", comment: "translators: accessibility text. empty post title.")
+let string70 = NSLocalizedString("Post title. %s", comment: "translators: accessibility text. %s: text content of the post title.")
+let string71 = NSLocalizedString("Link inserted", comment: "")
+let string72 = NSLocalizedString("Link text", comment: "")
+let string73 = NSLocalizedString("Add link text", comment: "")
+let string74 = NSLocalizedString("Translate", comment: "")
+let string75 = NSLocalizedString("hello", comment: "")
+let string76 = NSLocalizedString("%d banana", comment: "")
+let string76Plural = NSLocalizedString("%d bananas", comment: "")
+let string77 = NSLocalizedString("hello %s", comment: "")
+let string78 = NSLocalizedString("cheeseburger", comment: "")
+let string79 = NSLocalizedString("%d cat", comment: "")
+let string79Plural = NSLocalizedString("%d cats", comment: "")
```

Comment:
Is there a way to have those grouped so glotpress knows `string79Plural` is the plural version of `string79`? Not sure who to ping to answer this 🤔 Any idea @hypest ?

Reply:
We don't have support for plurals yet 😞 https://github.com/wordpress-mobile/WordPress-iOS/issues/6327

Reply:
I see. So I guess it's not worth sending those as extra strings for translations as they won't be picked up by glotpress as the plural version of other strings 🤔 Yeah in that case we should probably remove those